### PR TITLE
Feature/alph amount history

### PR DESCRIPTION
**Pull request by tdroxler**
Re-opening a cleaner version and reviewable PR of https://github.com/alephium/explorer-backend/pull/456

This PR is released as https://github.com/alephium/explorer-backend/releases/tag/v1.13.5-rc3

### CODE REVIEW

**Review by simerplaha — COMMENTED** [review]
Hey Thomas, I couldn't locate any test cases for the new functions, queries or endpoints added here.

Are they somewhere I'm not looking?

**Comment by tdroxler** [comment]
> Hey Thomas, I couldn't locate any test cases for the new functions, queries or endpoint added here.
> 
> Are they somewhere I'm not looking?

damn :sweat_smile: I was so focus on the feature I totally forgot them. Adding them now!

File: app/src/main/scala/org/alephium/explorer/web/AddressServer.scala [hunk 1]
```diff
@@ -45,6 +46,11 @@ class AddressServer(transactionService: TransactionService,
 
   val groupNum = groupSetting.groupNum
 
+  // scalastyle:off magic.number
+  private val maxHourlyTimeSpan = Duration.ofDaysUnsafe(7)
+  private val maxDailyTimeSpan  = Duration.ofDaysUnsafe(3650)
```
Comment (tdroxler) [inline app/src/main/scala/org/alephium/explorer/web/AddressServer.scala:51]: should be `365`, a left over of while I was testing and did want to be blocked by this.

**Review by polarker — COMMENTED** [review]
LGTM, but test is missing, so I am not able to review edge cases.
